catalog/core/catalog-core-solr/src/main/java/ddf/catalog/source/solr/SolrFilterDelegate.java

```diff
@@ -342,9 +342,15 @@ public SolrQuery propertyIsEqualTo(String propertyName, String literal, boolean
     String searchPhrase = QUOTE + escapeSpecialCharacters(literal) + QUOTE;
 
     if (searchPhrase.contains(SOLR_WILDCARD_CHAR)
-        || searchPhrase.contains(SOLR_SINGLE_WILDCARD_CHAR)
-        || Metacard.ANY_TEXT.equals(propertyName)) {
+        || searchPhrase.contains(SOLR_SINGLE_WILDCARD_CHAR)) {
       return new SolrQuery(wildcardSolrQuery(searchPhrase, propertyName, true));
```

Comment:
✏️ Nit: Can `wildcardSolrQuery` method be refactored so that part of that method doesn't need to be duplicated for this special case? I think that might make it easier to see the differences between `propertyIsInProximityTo`, `propertyIsFuzzy`, `propertyIsLike`, and `propertyIsEqualTo`.